packages/components/addon/components/hds/copy/button/index.hbs

```diff
@@ -0,0 +1,16 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<Hds::Button
+  class="{{if this.isSuccess 'is-success'}}"
+  @size={{this.size}}
+  @isFullWidth={{@isFullWidth}}
+  @text={{@text}}
+  @icon="{{if this.isSuccess 'clipboard-checked' 'clipboard-copy'}}"
+  {{on "click" this.copyCode}}
+  ...attributes
+  @color="secondary"
+  @iconPosition="trailing"
+  @clipboardText={{@clipboardText}}
```

Comment:
they would not be overwritten anyway, because they're _arguments_, not _attributes_ (`...attributes` spreads "attributes")

also, `@clipboardText` is not an argument for `<Hds::Button>` so should be removed